src/Components/ExperienceDetails.jsx

```diff
@@ -0,0 +1,99 @@
+import { useSelector } from 'react-redux';
+import { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { BiChevronRightCircle, BiChevronLeftCircle, BiGift } from 'react-icons/bi';
+import ExperienceIncluded from './ExperienceIncluded';
+import getRandomDiscount from '../logic/utils';
+
+function ExperienceDetails() {
+  const { experienceID } = useParams();
+
+  const [discount] = useState(getRandomDiscount({ start: 5, end: 50 }));
+
+  const { experiences } = useSelector((state) => state.experiences);
+
+  const selectedExperience = experiences.find((exp) => exp.id === parseInt(experienceID, 10));
+
+  const experienceSlug = selectedExperience?.name.replace(/\s/g, '-').toLowerCase();
+
+  const experienceReservationURL = `/${experienceSlug}/${experienceID}/new-reservation`;
+
+  const detailsIncluded = selectedExperience?.details === undefined ? '' : selectedExperience?.details;
+
+  return (
+    <div className="w-100">
+      <div className="experience-details w-100">
+        <div className="experience-image w-100">
+          <img src={selectedExperience?.image} alt={selectedExperience?.name} className="fluid-img" />
```

Comment:
[Optional] I recommended you decrease the size of the image on the desktop version, to `768px` and consider blocking the size at `600px`.

Reply:
@AndreaM2429 Thank you very much for taking the time to review my PR and suggesting these valuable changes. They have been added.